specification/dimensions/commitmentutilization.md

```diff
@@ -0,0 +1,37 @@
+# Commitment Utilization
+
+Commitment Utilization indicates whether a charge utilized a commitment during the charge period. Commitment Utilization applies to all commitments and commitment types applied to the current charge. This includes but is not limited to capacity commitments and commitment-based discounts represented by the Commitment Discount ID. Commitment Utilization is commonly used to identify wastage when a commitment has not been used and to calculate commitment utilization.
```

Comment:
**Commitment Utilization** is commonly used to identify wastage when a commitment has not been used and to calculate commitment utilization.
Commitment Utilization is used twice referring to different meaning

Reply:
Great catch! We're going to change the column name, so it's probably fine... Let's see how it looks when we update the column name.